### DISCUSSION TITLE
Consistent Language: Records versus Observations

### DISCUSSION THREAD

**Discussion by bms63** [pharmaverse/admiral#2066] [General]
This is in similar vein to our update around arguments and parameters.  Some background, our functions used arguments and parameters interchangeability in our function documentation and some users found this confusing.  We decided to only use the word `argument/s` in our function documentation and save `parameter/s` for discussing ADaMs - this required a minor update.

For this issue - we would like the word Records to be used rather than Observations as most CDISC documentation uses Records.

![image](https://github.com/pharmaverse/admiral/assets/10111024/fa46d116-6285-4851-8ee9-0f323c5dd0f0)

However, this issue might be a little tricky as some of our functions have `obs` in the function name, e.g. `derive_var_obs_number` and this update might have greater impact than arguments/parameters.

Any intrepid @pharmaverse/admiral willing to investigate this issue further?  I am looking basically for an assessment impact on the number of functions that would be impacted at the documentation level and if additional functions need to be re-named.  I think this research would take about an hour.  We would then make a decision on how/when to best implement this update.  Please respond in this post.

**Comment by manciniedoardo** [comment]
@bms63 personally I would put this discussion in the backlog by closing it until we get user feedback that this is actually a problem. After all "records" and "Observations" are synonymous anyway so I don't think it's worth trawling through documentation now in order to update.
